test/end-to-end/journal/trial_balance/trialBalance.page.js

```diff
@@ -27,15 +28,35 @@ function TrialBalancePage() {
     return gridRows.count();
   }
 
-  function openErrorModal() {
-    viewErrorButton.click();
+  function viewErrorList() {
+    return viewErrorButton.click();
+  }
+
+  function resetView(){
+    return resetButton.click();
+  }
+
+  function showAccountDetailInTransaction(n) {
+    const columnNumber = 5;
+
+    const row = grid
```

Comment:
I'm curious why you don't like to use [GridUtils](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/test/end-to-end/shared/GridUtils.js).  Is there some problem with that library?